libs/nss.c

```diff
@@ -115,11 +142,13 @@ int _nss_tcb_getspent_r(struct spwd *__result_buf,
 			closedir(tcbdir);
 			errno = saved_errno;
 			tcbdir = NULL;
+			*__errnop = ENOENT;
```

Comment:
I am not sure that our previous `errno` logic here was right, so maybe `ENOENT` is right here.

Reply:
Yes, as we cannot iterate over tcbdir for unknown reasons, so we should indicate "A necessary input file cannot be found."

Reply:
Given that in practice `__errnop` == `&errno`, assigning different values to `errno` and `*__errnop` doesn't make sense.